From 56015ad91302382d9be2338e0988bcb4a2455445 Mon Sep 17 00:00:00 2001
From: David Sniatkiwicz <davidsn@marvell.com>
Date: Mon, 25 Jun 2018 10:50:24 +0300
Subject: [PATCH 0762/1200] mvebu_mcbin-88f8040_defconfig: enable MVEBU_SAR on
 MACCHIATOBin

This option is required in order to read SoC information
using the sample at reset.

Change-Id: I07851784763399f888cb06fd55f568db546c9493
Signed-off-by: David Sniatkiwicz <davidsn@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/57241
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Igal Liberman <igall@marvell.com>
---
 configs/mvebu_mcbin-88f8040_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/mvebu_mcbin-88f8040_defconfig b/configs/mvebu_mcbin-88f8040_defconfig
index a074076317..719d1bd224 100644
--- a/configs/mvebu_mcbin-88f8040_defconfig
+++ b/configs/mvebu_mcbin-88f8040_defconfig
@@ -3,6 +3,7 @@ CONFIG_ARCH_MVEBU=y
 CONFIG_SYS_TEXT_BASE=0x00000000
 CONFIG_SYS_MALLOC_F_LEN=0x2000
 CONFIG_TARGET_MVEBU_ARMADA_8K=y
+CONFIG_MVEBU_SAR=y
 CONFIG_DEFAULT_DEVICE_TREE="armada-8040-mcbin"
 CONFIG_SMBIOS_PRODUCT_NAME=""
 CONFIG_DEBUG_UART=y
-- 
2.22.0

